.github/workflows/catalyst.yml

```
@@ -0,0 +1,33 @@
+name: 🐧 Catalyst
+
+on: [push, pull_request]
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.head_ref }}-insituvis-catalyst
+  cancel-in-progress: true
+
+jobs:
+  catalyst:
+    name: Catalyst
+    runs-on: ubuntu-22.04
+    if: github.event.pull_request.draft == false
+    env:
+      CXX: g++
+      CC: gcc
+      CMAKE_PREFIX: "/opt/conduit;/opt/catalyst"
```

@ChristosT did you mean to write here
```suggestion
CMAKE_PREFIX_PATH: "/opt/conduit:/opt/catalyst"
```
?

---

Posting this to #4067